### ISSUE TITLE
TODO and FIXME comment rules (S1134 & S1135) generate false positives, e.g. on "Au_toDo_wn"

### ISSUE THREAD

**Issue by dbolkensteyn** [SonarSource/sonarlint-vs#743]
Such a comment should not trigger an issue:
```
// AutoDown
```

This comment doesn't contain the word `TODO`, even if it does contain that character sequence.

**Comment by dbolkensteyn** [comment]
Issue found while working on #740
